lib/records/fhir.rb

```diff
@@ -240,6 +241,84 @@ def self.immunization(imm, fhir_record, patient, encounter)
         fhir_record.entry << entry
       end
 
+      def self.careplans(plan, fhir_record, patient, encounter)
+        careplanData = CAREPLAN_LOOKUP[plan['type']]
+        reasonCode = COND_LOOKUP[plan['reason']][:codes]['SNOMED-CT'][0]
+        reason = fhir_record.entry.find{|e| e.resource.is_a?(FHIR::Condition) && e.resource.code.coding.find{|c|c.code==reasonCode} }
+        careplan = FHIR::CarePlan.new({
+          'subject' => {'reference'=> "Patient/#{patient.fullUrl}"},
```

Comment:
Is this right?  Unless `#{patient.fullUrl}` resolves only to an ID, then I don't think this will be valid.